Azure/azure-cli
# [Security] Add adaptive_application_controls commands

**Description**  
[Security] Add adaptive_application_controls commands

Add adaptive_application_controls commands, helps you control which applications can run on your Azure and non-Azure machines (Windows and Linux)

**Testing Guide**  
Users can get a list, or a single adaptive application control groups:

az adaptive_application_controls list
az security adaptive_application_controls show --group-name GROUP1

Tests are included, use:  azdev test security test_adaptive_application_controls --live
---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [*] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [*] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).
